crates/nu-cli/src/commands/history.rs

```diff
@@ -64,9 +65,30 @@ impl Command for History {
             }
 
             if clear {
-                let _ = std::fs::remove_file(history_path);
-                // TODO: FIXME also clear the auxiliary files when using sqlite
-                Ok(PipelineData::empty())
+                let mut files_to_remove = Vec::from([history_path.clone()]);
+                if let HistoryFileFormat::Sqlite = engine_state.config.history_file_format {
+                    files_to_remove.extend_from_slice(&[
+                        history_dir.join("history.sqlite3-shm"),
+                        history_dir.join("history.sqlite3-wal"),
+                    ])
+                }
+                let (number_errors, message) = files_to_remove
+                    .iter()
```

Comment:
do you think there might be more file to remove here?

i mean i kinda like this iterator and the accumulation of the errors to get the nice error i show above :ok_hand: 
but it makes the code a bit more involved at the same time when the iterator only has two elements :yum:

Reply:
It has either 1 (txt) or 3 (sqlite) files. How would you simplify this? :)